src/Admin/AdherentAdmin.php

```diff
@@ -241,92 +227,66 @@ protected function configureFormFields(FormMapper $formMapper)
+                ->add('description', TextareaType::class, [
+                    'label' => 'Biographie',
                     'required' => false,
-                    'mapped' => false,
-                    'help' => 'Laisser vide si l\'adhérent n\'est pas membre du Conseil.',
+                    'attr' => ['class' => 'content-editor', 'rows' => 20],
                 ])
-                ->add('boardMemberRoles', 'sonata_type_model', [
-                    'expanded' => true,
-                    'multiple' => true,
-                    'btn_add' => false,
-                    'class' => Role::class,
-                    'mapped' => false,
-                    'help' => 'Laisser vide si l\'adhérent n\'est pas membre du Conseil.',
+                ->add('twitterPageUrl', UrlType::class, [
+                    'label' => 'Page Twitter',
+                    'required' => false,
+                    'attr' => [
+                        'placeholder' => 'https://twitter.com/alexandredumoulin',
+                    ],
                 ])
-            ->end()
-            ->with('Circonscription du député', ['class' => 'col-md-6'])
-                ->add('managedDistrict', 'sonata_type_model', [
-                    'label' => 'Nom de la circonscription du député',
-                    'by_reference' => false,
-                    'btn_add' => false,
+                ->add('facebookPageUrl', UrlType::class, [
+                    'label' => 'Page Facebook',
                     'required' => false,
+                    'attr' => [
+                        'placeholder' => 'https://facebook.com/alexandre-dumoulin',
+                    ],
                 ])
             ->end()
-            ->with('Coordinateur', [
-                'class' => 'col-md-6',
-                'description' => 'Laisser vide si l\'adhérent n\'est pas coordinateur. Utiliser les codes de pays (FR, DE, ...) ou des préfixes de codes postaux.',
-            ])
-                ->add('coordinatorCitizenProjectArea', CoordinatorManagedAreaType::class, [
-                    'label' => 'coordinator.label.codes.cp',
-                    'sector' => CoordinatorAreaSectors::CITIZEN_PROJECT_SECTOR,
-                ])
+            ->with('Responsabilités locales', ['class' => 'col-md-6'])
                 ->add('coordinatorCommitteeArea', CoordinatorManagedAreaType::class, [
                     'label' => 'coordinator.label.codes.committee',
                     'sector' => CoordinatorAreaSectors::COMMITTEE_SECTOR,
                 ])
+                ->add('managedArea', ReferentManagedAreaType::class, [
+                    'label' => false,
+                    'required' => false,
+                ])
+            ->add('procurationManagedAreaCodesAsString', TextType::class, [
+                'label' => 'coordinator.label.codes',
+                'required' => false,
+                'help' => 'Laisser vide si l\'adhérent n\'est pas responsable procuration. '.
```

Comment:
Extra white space at the end of the string.

What about using `"` instead of `'` to avoid backslashes to escape all `'` characters ?

Reply:
fixed